pkg/controller/build/build_controller.go

```diff
@@ -39,6 +39,27 @@ const succeedStatus string = "Succeeded"
 const namespace string = "namespace"
 const name string = "name"
 
+// error message
+const (
+	// validate secrets
+	ListSecretInNamespaceFailed string = "001: listing secrets in namespace %s failed"
```

Comment:
we should not mix the error code with the message, the message pls keep it the old way or find a different alternative, the constants should be pure error code

Reply:
BTW, the error code should not start as `00x`, it is better to start as `B001`, because in future we should have more error code, like BuildRun(BR) or BuildStrategy(BS), etc....

Reply:
@zhangtbj pls add a proper PR review for this

Reply:
Done for both requirements!